core-bundle/src/Resources/contao/modules/ModulePersonalData.php

```diff
@@ -121,6 +121,9 @@ protected function compile()
 		$objVersions->setEditUrl('contao/main.php?do=member&act=edit&id=%s&rt=1');
 		$objVersions->initialize();
 
+		$arrSubmitted = array();
+		$arrFiles = array();
+
```

Comment:
This one puzzles me, because it means that the `updatePersonalData` hook cannot have worked. `$arrSubmitted` only contained the last field before the change. 🤔

Reply:
Might be related to https://github.com/contao/contao/pull/4364 which is very new :)

Reply:
Ah, you are right. Never mind then. 👍